### PR TITLE
Prints CombineConstraint in a nicer way

### PR DESCRIPTION
**Pull request by topnessman**
1. For CombineConstraint, print result as:
1 = 2 ▷ 3 
Old way of printing is confusing : 1 = 3 + 2, in which 2 is the receiver, and the "+" sign seems not vivid.

### CODE REVIEW

File: src/checkers/inference/model/serialization/ToStringSerializer.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public String serialize(CombineConstraint constraint) {
         boolean prevShowVerboseVars = showVerboseVars;
         showVerboseVars = false;
         String result = indent(constraint.getResult().serialize(this) + " = ( "
-                + constraint.getDeclared().serialize(this) + " + " + constraint.getTarget() + " )");
+                + constraint.getTarget().serialize(this) + " \u25B7 " + constraint.getDeclared().serialize(this) + " )");
```
Comment (wmdietl) [inline src/checkers/inference/model/serialization/ToStringSerializer.java]: Can you add a comment what symbol `\u25B7` is?

File: src/checkers/inference/model/serialization/ToStringSerializer.java [hunk 2]
```diff
@@ -194,9 +194,6 @@ public String serialize(RefinementVariableSlot slot) {
     @Override
     public String serialize(ConstantSlot slot) {
         StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append("( ");
-        stringBuilder.append(slot.getId());
```
Comment (wmdietl) [inline src/checkers/inference/model/serialization/ToStringSerializer.java]: Isn't it useful to see the ID of a ConstantSlot? Or is this change to just show the constant, without the ID?

Comment (topnessman) [inline src/checkers/inference/model/serialization/ToStringSerializer.java]: Isn't it useful to see the ID of a ConstantSlot? 
-> Sometimes useful: If we want to see what IDs are assigned to ConstantSlots, it's useful; But if we want to see the constraint with ConstantSlot, having its ID increases difficulty of reading.
Or is this change to just show the constant, without the ID?
Yes, ConstantSlot's ID will not be printed anymore.

Comment (topnessman) [inline src/checkers/inference/model/serialization/ToStringSerializer.java]: Totally removing the id from the print result of ConstantSlot seems not proper. Maybe we shouldn't touch this serialize  ConstantSlot method.

**Pull request merged**